components/org.apache.stratos.rest.endpoint/src/main/java/org/apache/stratos/rest/endpoint/api/StratosApiV41Utils.java

```diff
@@ -127,10 +127,10 @@ public static void addCartridge(CartridgeBean cartridgeBean) throws RestAPIExcep
                         cartridgeBean.getType()));
             }
 
-            for(PortMappingBean portMapping : cartridgeBean.getPortMapping()) {
-                if(StringUtils.isBlank(portMapping.getName())) {
+            for (PortMappingBean portMapping : cartridgeBean.getPortMapping()) {
+                if (StringUtils.isBlank(portMapping.getName())) {
                     portMapping.setName(portMapping.getProtocol() + "-" + portMapping.getPort());
-                    if(log.isInfoEnabled()) {
+                    if (log.isInfoEnabled()) {
```

Comment:
Seems you have fixed only formatting issue here. Anyway, default carbon log level is set to "INFO", hence checking isInfoEnabled() introduce additional cost if it's same with org.apache.stratos too. Better check it and remove.